src/main/java/org/ggp/base/util/game/LocalGameRepository.java

```diff
@@ -226,7 +226,10 @@ private static int getMaxVersionForDirectory(File theDir) {
                 } else if (theFile.getName().endsWith(".png")) {
                     // TODO: Handle other binary formats?
                     return readBinaryFile(theFile);
-                } else if (theFile.getName().endsWith(".xsl")) {
+                } else if (theFile.getName().endsWith(".jpg")) {
+                    return readBinaryFile(theFile);
+                } else if (theFile.getName().endsWith(".xsl"))
+                {

```

Comment:
Sam,  are you referring to the brace indent style?  I thought my last commit did it correctly but I made several mistakes when doing my first pull request so that might not be the case.  Sorry about that.